components/places/src/db/schema.rs

```diff
@@ -113,6 +122,15 @@ const CREATE_TRIGGER_AFTER_INSERT_ON_PLACES: &str = "
     END
 ";
 
+const CREATE_TRIGGER_MOZPLACES_AFTERDELETE: &str = "
+    CREATE TEMP TRIGGER moz_places_afterdelete_trigger
+    AFTER DELETE ON moz_places
```

Comment:
Should we add to our `AFTER INSERT` trigger on `moz_places` to delete the GUID from `moz_places_tombstones`, so that we'll never end up with a GUID in both?